cli/cdsctl/version.go

```diff
@@ -30,43 +29,30 @@ func version() *cobra.Command {
 }
 
 func versionRun(v cli.Values) error {
-	var apiVersion *sdk.Version
-	var err error
-	if client != nil {
-		apiVersion, err = client.Version()
-	} else {
-		err = errors.New("no configuration file found")
+	format := v.GetString("format")
+	if format == "" {
+		fmt.Println(sdk.VersionString())
```

Comment:
with the default "" format the string: `CDS %s version:%s os:%s architecture:%s git.hash:%s build.time:%s db.migrate:%s` is not very clear for the user.

i would prefer to display:
* version of the CLI (with architecture, keychain enable, etc...)
* URL version of the connected API (if available)

something like `docker version`

Reply:
done
```bash
$ cdsctl version --format json
{"cdsctl":{"version":"snapshot","architecture":"amd64","os":"darwin","git_hash":"","build_time":"","keychain":true},"api":{"version":"snapshot","architecture":"amd64","os":"darwin","git_hash":"","build_time":""}}
 
$ cdsctl version
cdsctl:
  version: snapshot
  architecture: amd64
  os: darwin
  git_hash: ""
  build_time: ""
  keychain: true
api:
  version: snapshot
  architecture: amd64
  os: darwin
  git_hash: ""
  build_time: ""